### PR TITLE
DOP-4040: bump chatbot to v0.0.6

### PR DESCRIPTION
**Pull request by schmalliso**
### Stories/Links:

DOP-4040

### Current Behavior:

[Earlier staging](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/newhomepagewhodis/landing/allison/DOP-4024/)

### Staging Links:

[Staging](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/newhomepagewhodis/landing/allison/master/index.html)

### CODE REVIEW

File: src/components/ChatbotUi.js [hunk 1]
```diff
@@ -55,9 +55,10 @@ const StyledChatBotUiContainer = styled.div`
   width: 100%;
   background: ${palette.white};
   box-shadow: 0px 2px 3px 0px rgba(0, 0, 0, 0.1);
+  margin-left: -32px;
```
[the remark this answers is not in the document]
Comment (schmalliso) [reply src/components/ChatbotUi.js:58]: This is a much better solution. Always you are as santa to us.

[the remark this answers is not in the document]
Comment (schmalliso) [reply src/components/ChatbotUi.js]: Updated!

File: src/components/ChatbotUi.js [hunk 2]
```diff
@@ -72,21 +72,13 @@ const StyledChatBotUiContainer = styled.div`
...
 const LazyChatbot = lazy(() => import('mongodb-chatbot-ui'));
 
 const ChatbotUi = ({ template }) => {
   return (
     <StyledChatBotUiContainer data-testid="chatbot-ui" template={template}>
       {/* We wrapped this in a Suspense. We can use this opportunity to render a loading state if we decided we want that */}
-      <Suspense
-        fallback={<Skeleton wrapper={SkeletonWrapper} borderRadius={SKELETON_BORDER_RADIUS} width={771} height={82} />}
-      >
+      <Suspense fallback={<Skeleton borderRadius={SKELETON_BORDER_RADIUS} width={771} height={82} />}>
```
Comment (seungpark) [inline src/components/ChatbotUi.js:81]: would remove the width to have it adjust to mobile on load. not sure on setting an exact height..
```suggestion
      <Suspense fallback={<Skeleton borderRadius={SKELETON_BORDER_RADIUS} height={82} />}>
```

**Review by rayangler — APPROVED** [review]
Alignment LGTM. 

I noticed that some top and bottom margins for the chatbot have been changed as well due to the version bump (the container looks taller compared to previous staging links). But I will leave whether that needs to be changed up to the discretion of the design team!

**Review by seungpark — APPROVED** [review]

File: src/components/ChatbotUi.js [hunk 3]
```diff
@@ -77,7 +78,7 @@ const ChatbotUi = ({ template }) => {
   return (
     <StyledChatBotUiContainer data-testid="chatbot-ui" template={template}>
       {/* We wrapped this in a Suspense. We can use this opportunity to render a loading state if we decided we want that */}
-      <Suspense fallback={<Skeleton borderRadius={SKELETON_BORDER_RADIUS} width={771} height={82} />}>
+      <Suspense fallback={<Skeleton borderRadius={SKELETON_BORDER_RADIUS} height={82} />}>
         <LazyChatbot serverBaseUrl={CHATBOT_SERVER_BASE_URL} />
```
Comment (nlarew) [inline src/components/ChatbotUi.js:82]:
```suggestion
        <LazyChatbot
          serverBaseUrl={CHATBOT_SERVER_BASE_URL}
          suggestedPrompts={[
            "How do you deploy a free cluster in Atlas?",
            "How do you import or migrate data into MongoDB Atlas?",
            "Get started with MongoDB",
            "Why should I use Atlas Search?",
          ]}
        />
```

We need to define the suggested prompts client side

**Pull request merged**